### ISSUE TITLE
Support for no_std / libcore 

### ISSUE THREAD

**Issue by kevinaboos** [andersk/enum_primitive-rs#9]
This crate is ideal for use within an OS (to easily use C-style enums and defines), but it seems to require the std crate. Is there an option like a no_std feature to disable the std library and instead use just libcore? If not, I'd like to make that a feature request. 

If that's not possible, my apologies, I am relatively new to Rust.

**Comment by skull-squadron** [comment]
num-traits is moving to no_std too, so it would be a good idea. I'm also trying to use it in a no_std project.

**Comment by MrBuddyCasino** [comment]
Conjuring enums from ints is something thats often needed in embedded projects. No-Std would be great!

**Comment by repi** [comment]
Agreed, just ran into this today as well and was surprised that enumprimitive didn't support no_std
